### PR TITLE
Add vp9enc tool for testing driver

### PR DESCRIPTION
**Pull request by pengche1**
Signed-off-by: peng.chen <peng.c.chen@intel.com>

### CODE REVIEW

**Comment by yakuizhao** [comment]
Please add the correct license for vp9enc.c.

**Comment by pengche1** [comment]
@yakuizhao , sure, thanks!

File: encode/Makefile.am [hunk 1]
```diff
@@ -63,6 +63,13 @@ svctenc_LDADD		= \
 	$(top_builddir)/common/libva-display.la \
 	-lpthread -lm
 
+vp9enc_SOURCES     = vp9enc.c
+vp9enc_CFLAGS      = -I$(top_srcdir)/common -g
+vp9enc_LDADD       = \
+        $(LIBVA_LIBS) \
+    $(top_builddir)/common/libva-display.la \
+    -lpthread -lm
```
Comment (uartie) [inline encode/Makefile.am]: Please fix the tab formatting here.  It should be single hard-tab.

Comment (pengche1) [reply encode/Makefile.am]: sure, thanks

**Pull request merged**